src/components/pages/Login/LoginContainer.js

```diff
@@ -25,7 +25,6 @@ const LoginContainer = () => {
         registration: true,
       },
       // turning this feature on allows your widget to use Okta for user registration
-      logo: 'path-to-your-logo',
       // add your custom logo to your signing/register widget here.
```

Comment:
Removing the property is enough to warrant the removal of the code comment below as well.